src/ServiceFabric/ServiceFabric/Commands/ServiceFabricCmdletBase.cs

```diff
@@ -453,23 +458,26 @@ protected CertificateBundle ImportCertificateToAzureKeyVault(string keyVaultName
             var fileContentEncoded = Convert.ToBase64String(clearBytes);
 
             WriteVerboseWithTimestamp(string.Format("Importing certificate to Azure KeyVault {0}", certificateName));
-            var certificateBundle = this.KeyVaultClient.ImportCertificateAsync(
-                CreateVaultUri(keyVaultName).ToString(),
-                certificateName,
-                fileContentEncoded,
-                clearPassword,
-                new CertificatePolicy
+
+
+            CertificateClient client = new CertificateClient(
```

Comment:
are you going to use this local variable or the class level KeyVaultCertificateClient? you should use the same pattern everywhere. I see this.KeyVaultCertificateClient getting used in some other places, but I don't think this.certificateClient is initialized. Are those paths working? or am i missing something